### ISSUE TITLE
Python3.9 does not work out-of-the-box

### ISSUE THREAD

**Issue by jenisys** [hrcorval/behavex#16]
**Describe the bug**
Currently, you cannot install this package for Python3.9 .
The reason for this is the description in `setup.py` where python3.9 is explicitly disabled.

**To Reproduce**
1. Create a virtual environment for Python3.9 and activate it.
2. Try to install `behavex`with `pip` which will fail

```bash
# — EXAMPLE FOR: UNIX platform with bash as shell
$ virtualenv -p python3.9 .venv_py39
$ source .venv_py39/bin/activate
$ pip install behavex
…
ERROR: Package ‚behavex‘ requires a different Python: 3.9.x not in ‚!=3.9.x,>=3‘
```

**Expected behavior**
Normally, I would expect that Python3.9 (and Python3.10, …) are supported.

**Desktop (please complete the following information):**
 - OS: macOS (but this is irrelevant; error is not platform related)

**Comment by hrcorval** [comment]
Thanks for bringing this up. The wrapper was not working with an earlier python 3.9 version. We tested that again with latest python 3.9 and 3.10 versions, and it is working properly.
Fixed in upcoming release (v1.5.4)

**Comment by hrcorval** [comment]
Fixed in Behavex 1.5.4: https://pypi.org/project/behavex/1.5.4/
